android/src/main/java/com/ahm/capacitor/camera/preview/CameraPreview.java

```diff
@@ -262,7 +264,6 @@ public void run() {
 
                     getBridge().getWebView().setBackgroundColor(Color.TRANSPARENT);
                     ((ViewGroup)getBridge().getWebView().getParent()).addView(containerView);
-                    getBridge().getWebView().getParent().bringChildToFront(getBridge().getWebView());
```

Comment:
This should not be deleted

Reply:
I think this line can be conditionally executed based on some param. like, {...., isFront:true}

Reply:
See toBack prop in cordova-plugin-camera-preview plugin https://github.com/cordova-plugin-camera-preview/cordova-plugin-camera-preview#methods